internal/cluster/validations/validations.go

```diff
@@ -301,6 +301,9 @@ func ValidateHTTPProxyFormat(proxyURL string) error {
 // of destination domain names, domains, IP addresses or other network CIDRs. A domain can be
 // prefaced with '.' to include all subdomains of that domain.
```

Comment:
pls add "Use * to bypass proxy for all destinations." to the function's description.